modules/packages/alpine/metadata.go

```diff
@@ -188,6 +190,16 @@ func ParsePackageInfo(r io.Reader) (*Package, error) {
 			if value != "" {
 				p.FileMetadata.Dependencies = append(p.FileMetadata.Dependencies, value)
 			}
+		case "provider_priority":
+			n, err := strconv.ParseInt(value, 10, 64)
+			if err == nil {
+				p.FileMetadata.ProviderPriority = n
+			}
+		case "replaces_priority":
+			n, err := strconv.ParseInt(value, 10, 64)
+			if err == nil {
+				p.FileMetadata.ReplacesPriority = n
```

Comment:
Did I miss something, or is the `ReplacesPriority` currently unused?

Reply:
Yes, it is unused indeed. It is a part of Alpine package metadata, so it makes sense to parse and store it alongside all the other package metadata, but there isn't anything else being done with it currently. Specifically, it looks like there's no field for `replaces_priority` in the APKINDEX file, so `ReplacesPriority` is not used when generating APKINDEX.

I guess `ReplacesPriority` could show up in some HTTP API request if there is one that returns package file metadata, or if one is added in the future.

Reply:
In that case, I recommend adding a comment to the field stating exactly that.

Reply:
I removed the field as it's not used in the index spec at the moment. We can add it if it is used in future.

https://wiki.alpinelinux.org/wiki/Apk_spec#APKINDEX_Format